packages/expo-updates/android/build.gradle

```diff
@@ -60,6 +60,10 @@ def getBoolStringFromPropOrEnv(String name, Boolean defaultValue) {
 // debug builds. (default false)
 def exUpdatesNativeDebug = getBoolStringFromPropOrEnv("EX_UPDATES_NATIVE_DEBUG", false)
 
+// If true, app is using custom code to initialize expo-updates, so default initialization code
+// will be disabled.
+def exUpdatesCustomInit = getBoolStringFromPropOrEnv("EX_UPDATES_CUSTOM_INIT", false)
```

Comment:
we should make this a property that you can set in gradle.properties / Podfile.properties.json as well

Reply:
that's covered in the `getBoolStringFromPropOrEnv`